config/http.conf

```
@@ -51,9 +51,16 @@ http {
     ''      $request_id;
   }
 
+  # See https://www.nginx.com/blog/websocket-nginx
+  map $http_upgrade $connection_upgrade {
+    default upgrade;
```

I don't know if this affects anything but it looks like it's `Upgrade` online. Other values are also have the first letter capitalised e.g. `WebSocket` ...

```suggestion
    default Upgrade;
```